src/Symfony/Bundle/WebProfilerBundle/Controller/ExceptionErrorController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\WebProfilerBundle\Controller;
+
+use Symfony\Component\ErrorRenderer\ErrorRenderer\HtmlErrorRenderer;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Profiler\Profiler;
+
+/**
+ * Renders the exception panel.
+ *
+ * @author Yonel Ceruto <yonelceruto@gmail.com>
+ */
+class ExceptionErrorController
```

Comment:
missing internal flag

Reply:
also a better name would be ExceptionPanelController

Reply:
See https://github.com/symfony/symfony/pull/32741